src/Verb.js

```diff
@@ -120,7 +120,7 @@ TinCan client library
 
                 if (this.display === null && typeof _downConvertMap[this.id] !== "undefined") {
                     this.display = {
-                        "und": _downConvertMap[this.id]
+                        "en": _downConvertMap[this.id]
```

Comment:
I'm not sure this does go away with 0.9 support. I've seen clients using this as a shortcut to get the ADL verbs for 1.0.x still. 

Using 'und' in language maps is a particular issue when "canonical" format is requested as the LRS uses language negotiation. If the user's browser doesn't request "*", it could be argued that the LRS is correct not to return "und" even if it's the only language available and most browsers do not request "*" by default. 

Whether or not content negotiation applies to all language maps or just activity definition is ambiguous in 1.0.2 but 1.0.3 is set to clarify that it does apply to all language maps: https://github.com/adlnet/xAPI-Spec/blob/1.0.3/xAPI.md#language-filtering-requirements-for-canonical-format-statements

Today, I don't think any LRS does have a canonical verb display, but the client could still request canonical statements perhaps for the canonical activity definition.